README.tfdoc.hcl

```
@@ -217,6 +217,79 @@ section {
             }
           }
         }
+
+        variable "audit_configs" {
+          type = object(audit_log)
+          description    = <<-END
+            List of audit logs settings to be enabled.
+          END
+          readme_example = <<-END
+            audit_configs = [
+              {
+                service = "allServices"
+                configs = [
+                  {
+                    log_type = "DATA_READ"
+                  },
+                  {
+                    log_type = "DATA_WRITE"
+                  },
+                  {
+                    log_type = "ADMIN_READ"
+                  }
```

```suggestion
                  {
                    log_type = "ADMIN_READ"
                  },
                  {
                    log_type = "ADMIN_WRITE"
                  },
```

Let's add a full example here

---

ADMIN_WRITE is not a valid log_type
Audit Logs have 3 valid log types; ADMIN_READ, DATA_READ and DATA_WRITE
https://cloud.google.com/logging/docs/audit/configure-data-access#configuration_overview